llarp/rpc/rpc_server.cpp

```diff
@@ -1,7 +1,7 @@
 #include "rpc_server.hpp"
 #include <llarp/router/route_poker.hpp>
 #include <llarp/constants/version.hpp>
-#include <nlohmann/json.hpp>
```

Comment:
may as well still include this explicitly, rather than implicitly, since we're directly using the `nlohmann::json` type in here.

Reply:
`llarp/util/json.hpp` pulls this in

Reply:
I assumed as much, but I still think it would be better to include nlohmann/json explicitly since we're using types from it directly, rather than abstracted via llarp/util/json

Reply:
we do that already.